packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -289,6 +300,91 @@ export const createSeatManager = (
           return zcfSeat;
         },
 
+        /**
```

Comment:
good doc

Reply:
I copied & cleaned up from the doc on the helper. Kudos to @erights.